README_CN.md

```diff
@@ -0,0 +1,137 @@
+<p align="center">
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="OceanBase Logo" src="images/logo.svg" width="50%" />
+    </a>
+</p>
+<p align="center">
+    <a href="https://github.com/oceanbase/oceanbase/blob/master/LICENSE">
+        <img alt="license" src="https://img.shields.io/badge/license-MulanPubL--2.0-blue" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/releases/latest">
+        <img alt="license" src="https://img.shields.io/badge/dynamic/json?color=blue&label=release&query=tag_name&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase%2Freleases%2Flatest" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="stars" src="https://img.shields.io/badge/dynamic/json?color=blue&label=stars&query=stargazers_count&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
```

Comment:
Looks like the `stars` url is broken, is there any change in the badge api? @a1iive

Reply:
Github's problem :(